packages/application-components/src/components/internals/tabular-page.tsx

```diff
@@ -58,7 +67,13 @@ const FormControlsContainer = styled.div`
 `;
 
 const CustomViewsSelectorWrapper = styled.div`
+  // Bear in mind the margins are dependant on the context the component
+  // is rendered in. For example, when rendered in a custom view, its panel
+  // already contains default paddings, so we don't need to add margins here.
```

Comment:
Move comments outsides of CSS?

Reply:
I forgot about that one. Sorry.

Updated here: 64f6bef6a0578c2b403344243f5b566d520f363b